### ISSUE TITLE
Bulk Upload and Blind Minting

### ISSUE THREAD

**Issue by wwitkdqsck** [zeromintapp/community#11]
Just a Suggestion - >
The Goal:Create Similar Blind Minting Like Stupid Zombies Club, But On the Zeromint Website Itself.

Will Update the Details soon!

**Comment by treeder** [comment]
Just to flesh this out a bit, I think there's two parts.

1) Bulk upload via JSON file (this way would make them all available and viewable)
2) Option to bulk upload like 1, but only mint them randomly once purchased (or could just be sequential?) But user's can't see them until purchased. 

That sound about right?

**Comment by wwitkdqsck** [comment]
Yes @treeder  Ur Exactly Right!
 
When Users Upload the Images (must follow this number format ) 1.png,2.png,...,N.png
Same goes to the JSON files(must follow Zeromint format (the content of the file ) ) - 1.json,2.json...,N.json
[json.zip](https://github.com/zeromintapp/community/files/7840870/json.zip)

After they Set The Directories To Upload Then they can chose Option 1 or 2 above .

**Comment by treeder** [comment]
Let's use your GoBulls as a test for this. I believe the bulk import is already waiting for review, so we'd have to add the blind mint part to it then you could use your json file to bulk import.

**Comment by wwitkdqsck** [comment]
> Let's use your GoBulls as a test for this. I believe the bulk import is already waiting for review, so we'd have to add the blind mint part to it then you could use your json file to bulk import.

Sure ! Not a problem at all !
Will Update Soon.

**Comment by wwitkdqsck** [comment]
Okay @treeder @r-gochain  already complete Generating the random GoBulls images and Json files,The json file got 2 parts (a) full metadata in 1 file ,(b)seperate for each nfts   What should i do next ?
BTW ,once the user choose the DIrectories above this post ,if they choose Option 2 they will need to Upload another file (Preview file ( can be video or image ) ) must follow Zeromint Dimensions and Standards (file size)
[GoBulls-JSON.zip](https://github.com/zeromintapp/community/files/7847667/GoBulls-JSON.zip)

**Comment by treeder** [comment]
Might want to share that zip with us in private?

**Comment by treeder** [comment]
Preview image: Kind of like the stupid zombies one ya?  https://stupidzombies.club/

![image](https://user-images.githubusercontent.com/75826/148972048-8118289e-61fd-4832-8ada-2664962b869f.png)

**Comment by wwitkdqsck** [comment]
> Might want to share that zip with us in private?
In Private ? by email?

Yes ,that could work..i thought of using a video a Preview of all Gobulls 1024 nfts like this one 
[GoBears Preview ](https://zeromint.com/nfts/96E9h3SAXA4w7pHKPNby7)
[GoBulls Zeromint Page](https://zeromint.com/users/8KGCzGaIXScrIqUr1ohbHHiYqnf2)

**Comment by wwitkdqsck** [comment]
Hello guys,@treeder @r-gochain

For the option 2 (blind minting) above, can the owner / creator set the maximum nfts that can be purchase by a each wallet address by Chosing an Option(Input Field)? 
Max Number Per Address =

**Comment by treeder** [comment]
I don't think so, it's really impossible to limit what people can buy since they can just use a different address easily and buy more.

**Comment by wwitkdqsck** [comment]
Hi @treeder what i mean is each Zeromint account,if they create another account is okay...Meaning Each Zeromint Account can buy a certain Max Number of NFTs for the Blind Minting...you see i'm going to set the price very low...thats why...
###Just a Suggestion If Cant Be Done Its Okay,Will stick to the default.

**Comment by wwitkdqsck** [comment]
Okay Guys @r-gochain @treeder , 
After The User/Owner Upload The Files and Chooses The Options
When the Click Submit ->Zeromint Will Create 2 Folders on the User/Owner Folder -> (images) and (json) with all the files stored in it. ->Like This
[http://gobears.rf.gd/json/](http://gobears.rf.gd/json/)
[http://gobears.rf.gd/images/](http://gobears.rf.gd/images/)

I tried with TestNet Got Error -> Probably because its http not https->
[GoBearsTest](https://testnet-explorer.gochain.io/tx/0xb5402e920741bf4d1183995d5d936c1beb9780e87dc7b22dc6e883ea72ae9a71)

Will This Way Work Guys ?




![erro1](https://user-images.githubusercontent.com/93308499/149895625-025dc80b-ae1f-4560-8883-683dcd76ccc0.png)

**Comment by gobullsclub** [comment]
okay it works ,i tried hosting the files on github ->

[images](https://gobullsclub.github.io/images/1.png)
[json](https://gobullsclub.github.io/json/1.json)

[GoBullsTest](https://explorer.gochain.io/token/0x3943C75140d3C8Bf7538F31FE588cFc6a6cc7E8C?addr_tab=transactions)

**Comment by treeder** [comment]
Wait, so you're pre-minting them?

**Comment by gobullsclub** [comment]
No @treeder ,i'm planning to do it exactly how you did with stupid zombies club ,but on zeromint website itself ...i'm just suggesting the folders options above post...can it work ? Store the JSON and IMAGES to the Zeromint User/Owner Folder on Zeromint

**Comment by gobullsclub** [comment]
@r-gochain @treeder do you mean ,for the stupid zombies club ,the blind minting ,the images haven't been generated?till someone mints them ?

**Comment by treeder** [comment]
Hey @gobullsclub , just so we don't hold you up on this, maybe you should do it like you did for the bears while we work on this, then we can try this for your next one. This may take a couple of weeks.

**Comment by gobullsclub** [comment]
Okay Travis !! @treeder @r-gochain No problem at all ! i'll do it Like The Gobears! And Yes ,im planning to do another NFT Collection With Better Image and Standards ...So maybe that one i'll do the Bulk Minting Option !
Thanks Again!

**Comment by gobullsclub** [comment]
Hello  @treeder @r-gochain Should We close this issue? since we are moving Fully  towards Metamask (Like Opensea) ?
